Public/script.js

```diff
@@ -77,8 +106,8 @@ async function makeTravelPlan() {
 //   makeTravelPlan();
 
   searchBtn.addEventListener("click", async () => {
-    let countryName = countryInp.value;
-    await fetchAndDisplayCountryDetails(countryName);
-    await makeTravelPlan(countryName);
-    console.log(countryName);
+    // let countryName = countryInp.value;
```

Comment:
I had to remove countryName from the given parameter as within the even listener button is the function "fetchAndDisplayCountryDetails();" - and within that function, the getRandomCountry is also called.